sites/statics/server.js

```diff
@@ -0,0 +1,19 @@
+var path = require("path");
```

Comment:
This file may be better in the `site-utilities` folder which is meant to serve as shared assets between sites.

Reply:
Per discussion with @janechu, moving my statics into `site-utilities`